scripts/vendors/microsoft/configure.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+import argparse
+
+def main(github_pool, num_lists):
+  github_runner = None
+  test_lists = []
+  platforms = ["windows-x64", "windows-arm64", "linux-x64", "linux-arm64", "macos-x64", "macos-arm64"]
```

Comment:
Should this be hard-coded? Or perhaps should it be a parameter sent in with a default value of this?

Reply:
I am fine with it being hard coded. I dont think these will change that often. Then either way we would need a change request to update the python or YAML